lptree.pas

```diff
@@ -4403,6 +4431,21 @@ function TLapeTree_Operator.Compile(var Offset: Integer): TResVar;
     end;
   end;
 
+  function TryOperatorOverload(): TResVar;
+  begin
+    Result := NullResVar;
+    if (FOperatorType in OverloadableOperators) and RightVar.HasType() and
+       RightVar.HasType() and (not (FOperatorType in UnaryOperators)) then
```

Comment:
I guess. Didn't really wanna do that as I know I can be very forgetful, and that could result in adding "unary minus" to the OverloadableOperators list, at some time in the future, and that would blow up lape.